contracts/core/AuxiliaryBlockStore.sol

```diff
@@ -378,4 +441,47 @@ contract AuxiliaryBlockStore is BlockStore {
 
         isAncestor_ = currentCheckpoint.blockHash == _ancestor;
     }
+
+    /**
+     * @notice Stores the kernel observation for the dynasty number.
+     *         It also activates already reported kernel
+     *
+     * @param _blockHash The checkpoint that shall be finalized.
```

Comment:
?

Reply:
Ahh!!! sorry, I changed the logic and did not update the comments. 
Its updated now.